Jenkinsfile_CNP

```diff
@@ -334,46 +334,10 @@ def generateDefinitions() {
 
 def setFeatureFlags(environment) {
   if (environment =~ /preview|demo|perftest/) {
-    env.CONTACT_PARTIES_FEATURE = true
-    env.LINK_CASE_FEATURE = true
-    env.CASE_FLAGS_FEATURE = true
-    env.CASE_FILE_VIEW = true
-    env.DOCUMENT_MANAGEMENT_FEATURE = true
-    env.CREATE_BUNDLING_FEATURE = true
-    env.EDIT_BUNDLING_FEATURE = true
-    env.CLONE_BUNDLING_FEATURE = true
-    env.STITCH_BUNDLING_FEATURE = true
     env.SECURITY_CLASSIFICATION = true
-    env.MIGRATION_ENABLED = true
-    env.ACCESS_PROFILES_ENABLED = true
-    env.DSS_UPDATE_CASE_ENABLED = true
   } else if (environment =~ /aat/) {
-    env.CONTACT_PARTIES_FEATURE = true
-    env.LINK_CASE_FEATURE = true
-    env.CASE_FLAGS_FEATURE = true
-    env.CASE_FILE_VIEW = true
-    env.DOCUMENT_MANAGEMENT_FEATURE = true
-    env.CREATE_BUNDLING_FEATURE = true
-    env.EDIT_BUNDLING_FEATURE = true
-    env.CLONE_BUNDLING_FEATURE = true
-    env.STITCH_BUNDLING_FEATURE = true
     env.SECURITY_CLASSIFICATION = true
-    env.MIGRATION_ENABLED = true
-    env.ACCESS_PROFILES_ENABLED = true
-    env.DSS_UPDATE_CASE_ENABLED = true
   } else if (environment =~ /prod/) {
-    env.CONTACT_PARTIES_FEATURE = true
-    env.CASE_FILE_VIEW = false
-    env.DOCUMENT_MANAGEMENT_FEATURE = true
-    env.LINK_CASE_FEATURE = true
-    env.CASE_FLAGS_FEATURE = true
-    env.CREATE_BUNDLING_FEATURE = true
-    env.EDIT_BUNDLING_FEATURE = true
-    env.CLONE_BUNDLING_FEATURE = true
-    env.STITCH_BUNDLING_FEATURE = true
     env.SECURITY_CLASSIFICATION = false
-    env.MIGRATION_ENABLED = true
-    env.ACCESS_PROFILES_ENABLED = true
-    env.DSS_UPDATE_CASE_ENABLED = true
```

Comment:
Do we need to keep the feature flag for dss update case?

Reply:
No it has been enabled globally